packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -221,6 +228,30 @@ class AttestationRequestHandler {
 
     return attestation
   }
+
+  private async verifyPepperIfApplicable(): Promise<void> {
```

Comment:
Nit: This doesn't look like it needs to be async but I've been wrong before

Reply:
I think the bls verify function needs needs to be async

Reply:
It's always seemed strange to me that thresholdBls.verify often has an await in front of it even though it's synchronous. To my knowledge there isn't a good reason for this, but happy to keep this as is for now